packages/relay-compiler/bin/RelayCompilerBin.js

```diff
@@ -83,6 +84,32 @@ function getFilepathsFromGlob(
   });
 }
 
+function getLanguagePlugin(options: {language: string}): PluginInterface {
+  if (options.language === 'javascript') {
+    return RelayLanguagePluginJavaScript();
+  } else {
+    try {
+      // prettier-ignore
+      // $FlowFixMe
+      let languagePlugin = __non_webpack_require__( // eslint-disable-line no-undef
+        `relay-compiler-language-${options.language}`,
+      );
+      if (languagePlugin.default) {
+        languagePlugin = languagePlugin.default;
+      }
+      if (typeof languagePlugin === 'function') {
+        // For now a plugin doesn’t take any arguments, but may do so in the future.
+        return languagePlugin();
+      }
+    } catch (err) {}
```

Comment:
I see, so if it isn't webpack it just gets ignored here?

Reply:
No, this is meant to more gracefully handle a plugin name being misspelled or the plugin not conforming to the expectation of exporting a function.

Reply:
Improved this a little further.